### PR TITLE
Feature 3.5/swagger 3.25.1

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of #11530

### CODE REVIEW

**Comment by Simran-B** [comment]
Frank and Heiko have nothing against the change to "none" and I think it's for the better.

File: js/server/assets/swagger/index.html [hunk 1]
```diff
@@ -86,8 +86,8 @@
         ],
         layout: "StandaloneLayout",
         //#region ArangoDB-specific changes
-        // Expand the API documentation by default.
-        docExpansion: "list",
+        validatorUrl: null,
+        docExpansion: "none",
```
Comment (pluma) [inline js/server/assets/swagger/index.html:90]: See the other PRs. I'd prefer keeping this as `"list"` so we don't have to backport #11579 to every version.

Comment (Simran-B) [reply js/server/assets/swagger/index.html]: Restored to "list"

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10019/

http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/627/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**